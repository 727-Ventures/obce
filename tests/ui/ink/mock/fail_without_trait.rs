#[obce::mock]
impl () {
    fn method(&self) {}
}

fn main() {}
