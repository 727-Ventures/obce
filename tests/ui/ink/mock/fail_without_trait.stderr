error: impl marked as mocked should have a trait present
 --> tests/ui/ink/mock/fail_without_trait.rs:2:1
  |
2 | / impl () {
3 | |     fn method(&self) {}
4 | | }
  | |_^
